src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -518,12 +519,14 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
 
             if (noJekyll) FIO.touch(siteDir / ".nojekyll")
 
-            ghOps.commitDir(branch, commitMessage, siteDir).value.unsafeRunSync() match {
-              case Right(_) => log.info("Success committing files")
-              case Left(e) =>
+            ghOps
+              .commitDir(branch, commitMessage, siteDir)
+              .map(_ => log.info("Success committing files"))
+              .handleError { e =>
```

Comment:
`handleErrorWith` would be better here since there is `e.printStackTrace`, no?

Reply:
👍